Sources/MapboxNavigation/UserCourseView.swift

```diff
@@ -35,7 +35,7 @@ public extension CourseUpdatable {
 /**
  A view representing the user’s location on screen.
  */
-public class UserPuckCourseView: UIView, CourseUpdatable {
+open class UserPuckCourseView: UIView, CourseUpdatable {
```

Comment:
This change to UserPuckCourseView and UserHaloCourseView is worth mentioning in the changelog, small as it is.

Reply:
Thanks for the suggestion, I'll add changeLog list for it.